service/src/main/kotlin/io/provenance/api/domain/usecase/cee/execute/ExecuteContractBatch.kt

```diff
@@ -53,9 +66,7 @@ class ExecuteContractBatch(
                     provenanceService.buildContractTx(args.request.config.provenanceConfig, BatchTx(it)).let { tx ->
                         provenanceService.executeTransaction(args.request.config.provenanceConfig, tx, signer).let { pbResponse ->
                             responses.add(
-                                ContractExecutionResponse(
-                                    false,
-                                    null,
+                                SinglePartyContractExecutionResponse(
```

Comment:
I'm not sure if this matters, but the response from the transaction execution isn't necessarily a Single Part Contract Execution.. we're submitting any SignedResult whether it's single-party or multi-party. Should we maybe create a separate class for provenance transaction successful responses rather than reusing the p8e contract execution response classes?

Reply:
I suppose a more useful distinction to make here is if it's `pending` or `completed`. Reviewing the Contract Engine, it's only possible for one signature to be added on the initial execution of a contract, so it is safe here to mark this as single party since only single party contracts would return a SignedResult from this method.